plugins/modules/network_settings_intent.py

```diff
@@ -1786,10 +1818,7 @@ def update_network(self, config):
         site_name = config.get("network_management_details").get("site_name")
         result_network = self.result.get("response")[2].get("network")
         result_network.get("response").update({site_name: {}})
-        obj_params = [
-            ("settings", "settings"),
-            ("site_name", "site_name")
-        ]
+        obj_params = self.get_obj_params("Network")
```

Comment:
network_obj_params = self.get_obj_params("Network")

If it is already set, then we can directly use network_obj_params in verify instead of getting again..